README.md

```diff
@@ -1 +1,5 @@
-# .github
\ No newline at end of file
+![test](https://github.com/renovatebot/.github/workflows/test/badge.svg)
+
+# .github
+
+This repository contains the basic Renovate configuration that is used in most/all other Renovate repositories.
```

Comment:
I'm not sure if we use this repository for _all_ or only for _most_ repos?

Reply:
For all, other repos need to be updated 🙃